src/builder.rs

```diff
@@ -594,6 +594,11 @@ impl CpModelBuilder {
         Constraint(index)
     }
 
+    /// Delete all constraints.
+    pub fn del_all_cst(&mut self) {
```

Comment:
I prefer to not add this method: the cpp interface doesn't propose it, and it's quite dangerous in a real modeling situation, as you might think some variable still has their values, and is not really useful in practice as you can just create a new model.

Reply:
Ok